src/common/ZModal.tsx

```diff
@@ -18,12 +19,16 @@ const ZModal: React.FC<ZModalProps> = ({ children, type, open, onCancel, width,
     backgroundColor: darkModeStatus ? "rgba(0, 0, 0, 0.50)" : "rgba(87, 87, 87, 0.4)",
   };
 
+  const isMobileDevice = () => {
```

Comment:
umm, i don't see any change . any specific reason to add this check ? why no just keep it centered by default ?